### PR TITLE
Updated Integration Tests

### PR DESCRIPTION
**Pull request by meetagrawal09**
## Description

This PR tries to change the way we access functions from a package in integration tests.

## Review Time Estimate

- [ ] Immediately
- [ ] Within one week
- [X] When possible

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue) 
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] My name is in the list of CITATION.cff
- [ ] I have updated the CHANGELOG.md.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/data.atmosphere/inst/integrationTests/test.download.AmerifluxLBL.R [hunk 1]
```diff
@@ -1,19 +1,23 @@
 library(testthat)
+library(mockery)
+library(PEcAn.DB)
+library(PEcAn.logger)
+library(withr)
 
 test_download_AmerifluxLBL <- function(start_date, end_date, sitename, lat.in, lon.in) {
   # putting logger to debug mode
-  PEcAn.logger::logger.setUseConsole(TRUE, FALSE)
```
Comment (mdietze) [inline modules/data.atmosphere/inst/integrationTests/test.download.AmerifluxLBL.R:5]: In general I'm surprised by the need to add the library calls. I'm even more surprised that you need to actively remove the namespace info. Did you try a version that added the library calls without deleting the namespace?

Comment (meetagrawal09) [reply modules/data.atmosphere/inst/integrationTests/test.download.AmerifluxLBL.R:5]: Yes it does seem kind of strange. But I did try to add library calls and keep the namespaces but that too resulted in a failure while running the tests.

**Comment by meetagrawal09** [comment]
These tests work with all versions of `mockery` and can now be run using the command: 
```
 Rscript modules/data.atmosphere/inst/integrationTests/test.download.ERA5.R
 ```

File: modules/data.atmosphere/inst/integrationTests/test.download.AmerifluxLBL.R (hunk 1 above)
Comment (meetagrawal09) [reply modules/data.atmosphere/inst/integrationTests/test.download.AmerifluxLBL.R]: I tried out removing just the `PEcAn.DB` namespace and putting that to a library call. This works well.

**Review by mdietze — APPROVED** [review]

**Pull request merged**